frontend/src/features/mapSideBar/containers/MapSideBarContainer.tsx

```diff
@@ -80,6 +81,7 @@ const EditButton = styled(FaEdit)`
 const MapSideBarContainer: React.FunctionComponent<IMapSideBarContainerProps> = ({
   movingPinNameSpaceProp,
 }) => {
+  const formikRef = React.useRef<FormikValues>();
```

Comment:
I've seen this added a couple times now, what is this doing?

Reply:
Refs are a react concept, often use to allow parent components to interact with child components. https://reactjs.org/docs/refs-and-the-dom.html

Refs can be used with formik to allow parent components to access the form within a child component. This allows the parent component to check the state of the form, and initiate form actions, such as submitting the form, or validating the form.